# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: servo/components/style/properties/properties.mako.rs
# Commit: 6f62586a139b
# Full Hash: 6f62586a139b1fcc4817dddd924400958d89a6bb
# Author: Zach Hoffman <zach@zrhoffman.net>
# Date: 2023-12-18 17:17:57
# Regressor Bug: 1846516
# File Overlap Count: 2
# Description:
#   Bug 1846516 - [css-properties-values-api] Animate custom properties in a discrete way. r=firefox-style-system-reviewers,emilio
#   
#   This patch also updates the bug ID for a FIXME leftover from bug 1840478
#   to bug 1869476, since the same FIXME is added in D190758.
#   
# ==============================================================================

diff -r 729abedf9bda -r 6f62586a139b servo/components/style/properties/properties.mako.rs
--- a/servo/components/style/properties/properties.mako.rs	Mon Dec 18 09:24:08 2023 +0000
+++ b/servo/components/style/properties/properties.mako.rs	Mon Dec 18 09:24:08 2023 +0000
@@ -3254,7 +3254,7 @@
                 s
             }
             PropertyDeclarationId::Custom(name) => {
-                // FIXME(bug 1273706): This should use a stylist to determine
+                // FIXME(bug 1869476): This should use a stylist to determine
                 // whether the name corresponds to an inherited custom property
                 // and then choose the inherited/non_inherited map accordingly.
                 let p = &self.custom_properties;